src/util/sienna/helper_functions.jl

```diff
@@ -131,58 +143,7 @@ function line_type(line::DCLine) where {DCLine <: HVDCLineTypes}
     error("line_type isn't defined for $(typeof(line))")
 end
 
-"""
-    get_ts_values(ts::PSY.TimeSeriesData)
-
-Get the time series values.
-
-# Arguments
-
-  - `ts::TimeSeries`: Time series
-
-# Returns
-
-  - `Array{Float64}`: Time series values
-"""
-function get_ts_values(ts::PSY.TimeSeriesData)
-    error("get_ts_values isn't defined for $(typeof(ts))")
-end
-
-function get_ts_values(ts::TS) where {TS <: PSY.AbstractDeterministic}
-    if (typeof(ts) == PSY.DeterministicSingleTimeSeries)
-        forecast_vals = get_ts_values(ts.single_time_series)
-    else
-        forecast_vals = []
-        for it in collect(keys(PSY.get_data(ts)))
-            append!(
-                forecast_vals,
-                collect(values(PSY.get_window(ts, it; len=interval_len))),
-            )
-        end
-    end
-    return forecast_vals
-end
-
-function get_ts_values(ts::TS) where {TS <: PSY.StaticTimeSeries}
-    forecast_vals = values(PSY.get_data(ts))
-    return forecast_vals
-end
-
-"""
-    get_first_ts(ts::TS)
-
-Get time series and handle components with no time series (such as unavailable)
-"""
-function get_first_ts(ts::TS) where {TS <: Channel{Any}}
-    if isempty(ts)
-        return nothing
-    else
-        return first(ts)
-    end
-end
-function get_forecast_values(ts::Nothing)
-    return zeros(length(period_of_interest))
-end
+st_ts_key_filter_func = x -> isa(x, PSY.IS.StaticTimeSeriesKey)
```

Comment:
StaticTimeSeriesKey is a private IS object. What are you trying to achieve here?

Reply:
I am trying to check if the time series name user is specifying actually is associated with the component. Because we don't have get_time_series_names anymore, I am gettting all the time_series_keys filtering for only StaticTimeSeries keys and getting the names of them. Is that making sense?